README.md

```diff
@@ -1,140 +1,146 @@
    - MIME: application/vnd.lumenradio.net_rate
-   - Payload: 0 to 10 as an integer value.
+   - Payload: 8
 5. Update interval - seconds between measurement
    - MIME: application/vnd.lumenradio.update_interval
-   - Payload: 0 to 65534 as an integer value.
+   - Payload: 60
 6. Move threshold - Threshold value above which motion should be detected
    - MIME: application/vnd.lumenradio.move_theshold
-   - Payload: 0 to 127 as integer value.
-
-The current commissioning status can be read via NFC. The same set of records
-is available, except for network key.
-
-When commissioned, the current network status is also available.
-
-
-Flashing
---------------------
-First, add a license to the device on address 0x7F000 of length 0x1000, according to MiraOS documentation:
-
-https://docs.lumenrad.io/miraos/2.8.0/description/licensing/licensing_tool.html#signing-a-connected-device
-
-...or by flashing the license as a hex file if available.
-
-Second, flash the application to the device:
-
-```
-nrfjprog -f nrf52 --program mira-demo-nrf52832ble-os.hex --sectorerase --verify
-```
-
-Decommissioning
---------------
-
-To disable the node, use the same method as commissioning, but write:
-- PAN-ID = ffffffff
-- Encryption key = ffffffffffffffffffffffffffffffff
-- Rate = 255
-- Update interval = 65535
-
-Local monitoring of sensor values
----------------------------------
-
-When the device is commissioned, the sensor values is available via NFC as
-fields in the NDEF records. The values is available as user-readable text
-under the MIME type application/vnd.lumenradio.sensor.\*
-
-Sensor values is updated at an interval specified via update interval
-configuration
-
-Packet format for data
--------------
-
-The sensor sends an UDP packet at port 7338 to the root node of the network
-containing:
-
-- 16 bytes name of device, padded with zeroes. Note that a name with 16 bytes
-  may not have a null-byte termination
-- 40 bytes parent address
-- list of sensor values, 9 bytes each
-  - 1 byte type:
-    - 0 = no type
-    - 1 = Temperature
-    - 2 = Pressure
-    - 3 = Humidity
-    - 4 = Battery
-    - 5 = ETX
-    - 6 = Clock drift
-    - 7 = CO2
-    - 8 = Acc_x
-    - 9 = Acc_y
-    - 10 = Acc_z
-    - 11 = Move count
-    - 12 = seq num
-  - 4 bytes, MSB first, signed, sensor P value
-  - 4 bytes, MSB first, unsigned, sensor Q value
-
-The sensor values is transmitted as a rational value. To get the correct sensor
-value, calculate P/Q.
-
-This method makes it possible for the sensor to provide a proper range and
-resolution for presentation.
-
-With this format 11 sensor values can be added before fragmentation occurs.
+   - Payload: 64
 
-Receiving and presenting
------------------------
 
-An example script for receiving sensor updates, and publish to InfluxDB is
-provided as udp\_to\_influx.py
+When commisioning several Ruuvi tags, make sure to set a different name in step 1. All other parameters can remain the same.
```

Comment:
Perhaps it's good to mention that the Network PAN ID and Network encryption key need to be the same as what is set in the gateway configuration.

Reply:
The current README is now using the default values for the gateway. I was thinking about adding a "advanced configuration" section or similar at the end, and leave these details out of the "Quick setup" to keep it as short as possible. Would that make sense?

Reply:
Yeah, that would make sense I think.  
But I still think that it's good to emphasize that the keys should be the same on both the tags and the gateway. And possible also state that they should be changed.  
Even in the "Quick setup" section.

Reply:
done

Reply:
I think it looks good now!